rubygem-infoblox/rubygem-infoblox.spec

```diff
@@ -0,0 +1,84 @@
+# Generated from infoblox-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name infoblox
+
+Name: rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: A Ruby wrapper to the Infoblox WAPI
+Group: Development/Languages
+License: MIT
+URL: https://github.com/govdelivery/infoblox
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(faraday)
+Requires: rubygem(faraday_middleware)
```

Comment:
Ah, I ddin't noticed they are not in nonscl repos, I've added both to nonscl targets. I'm not sure why faraday is in core repos and faraday_middleware in plugins so I followed the pattern.